Bottom: fc40da3010945a7c8f4d29d5aefc1e48e8eb64f7
Top:    c6481b45d83d589d308c35225af46e102dce0f68
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-23 17:35:52 +0000

Refresh of add-hid_draw-h-to-makefile-am

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 269ee78..602ebaa 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -79,6 +79,7 @@ PCB_SRCS = \
 	heap.c \
 	heap.h \
 	hid.h \
+	hid_draw.h \
 	insert.c \
 	insert.h \
 	intersect.c \
